devops/demo/landing-page/webroot/index.html

```diff
@@ -5,6 +5,14 @@
         <link rel="stylesheet" href="/assets/css/demo.css">
     </head>
     <body>
+        <!-- Apache-licensed "content copy" icon from https://material.io/resources/icons/ -->
+        <svg style="display:none">
```

Comment:
In case you're curious, `xmlns` is [not required](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Element/svg) for inline SVGs.